examples/resources/artie_encryption_key/import.sh

```
@@ -0,0 +1,9 @@
+# Import an encryption key by using its UUID, which you can find by:
+# 1. Go to the pipeline overview page in the Artie UI (for a pipeline that uses this encryption key)
+# 2. Open the dropdown in the top right corner
+# 3. Select "View UUIDs" to see all related resource UUIDs
+terraform import artie_encryption_key.my_encryption_key <encryption_key_uuid>
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)